packages/core/test/JWeditor.test.ts

```diff
@@ -168,7 +168,66 @@ describe('core', () => {
                     );
                     expect(expectedCommands).to.eql(['command-all', 'command-pc']);
                 });
-                it('should load the config for keymap and override default keymap', () => {});
+                it('should load the config for keymap and override default keymap with priority', () => {
+                    const editor = new JWEditor();
+                    editor._platform = 'pc';
+                    editor.start();
+                    // todo: to remove when the corePlugin will be removed from editor.start().
+                    editor.eventManager.keymap._shortcutItems = [];
+                    editor.addPlugin(
+                        class A extends JWPlugin {
+                            shortcuts = [
+                                {
+                                    pattern: 'ctrL+a',
+                                    commandId: 'command-all',
+                                },
+                            ];
+                        },
+                    );
+                    editor.loadConfig({
+                        debug: true,
+                        shortcuts: [
+                            {
+                                pattern: 'ctrL+a',
+                                commandId: 'command-all',
+                            },
+                        ],
+                    });
+                    const priorityCommands = editor.eventManager.keymap._shortcutItems.map(
+                        item => item.priority,
+                    );
+                    expect(priorityCommands).to.eql([0, 1]);
+                });
```

Comment:
One of your tests (maybe you have one already, I don't know) should call `addPlugin` *after* calling `loadConfig` to check that the plugin shortcuts do not override the config ones. (Though that would not be possible if we use two keymaps like I suggested earlier.)